scripts/validate-runtime-release.mjs

```
@@ -0,0 +1,76 @@
+  /OPENCODE_VERSION\s*=\s*["']\d+\.\d+\.\d+["']|ARG OPENCODE_VERSION=\d+\.\d+\.\d+/,
+  "OpenCode pins must derive from release.json"
+);
+
+const dockerfile = readFileSync("packages/e2b-infra/e2b.Dockerfile", "utf8");
+assert.equal(
+  dockerfile.match(/__OPENCODE_VERSION__/g)?.length,
+  1,
+  "E2B Dockerfile must contain exactly one OpenCode version token"
+);
+
+console.log(`Runtime release metadata valid (OpenCode ${release.opencode_version})`);
```

ESLint

[error] 'console' is not defined.

(no-undef)